recipes/kaolin/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "kaolin" %}
+{% set version = "0.9.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIAGameWorks/kaolin/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7fa6b794d83fca9b4a1eb93b13ead60767b42d5a8a25162ee73be3cead1a8c8c
+
+build:
+  number: 0
+  skip: true  # [cuda_compiler_version == "None" or cuda_compiler_version == "10.2"]
+
+requirements:
+  build:
+    - cython
+    - cuda
+    - {{ compiler('cxx') }}
+    - python
+  host:
+    - pip
+    - python 
+  run:
+    - python 
```

Since this is dependent on `numpy` and it is linked to, would include it in the requirements here and use `pin_compatible` to make sure we get a version at runtime that is compatible with the one we built against

```suggestion
    - python
    - {{ pin_compatible('numpy') }}
```